subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/TestFilter.java

```diff
@@ -31,7 +31,7 @@
  *          //specific test method
  *          includeTestsMatching "org.gradle.SomeTest.someSpecificFeature"
  *
- *          //specific test method, use wildcard for packages
+ *          //specific test method, use wildcard
```

Comment:
❌ I think we should change this to show your new matcher instead. Wildcard shouldn't be necessary for this use case.